lib/ansible/modules/files/copy.py

```diff
@@ -174,17 +180,17 @@
     content: '# This file was moved to /etc/other.conf'
     dest: /etc/mine.conf
 
-- name: if follow is true, /path/to/file will be overwritten by contents of foo.conf
+- name: If follow is true, /path/to/file will be overwritten by contents of foo.conf
```

Comment:
I'd update the `name` line to match the parameter. So "If follow is set to 'yes', . . . ", since we have `follow: yes` below.

Reply:
Sure, and I have been doing that for some, but it was not my intention to review everything as that would take too much time :-)

Reply:
I mostly fix what I find on my path towards fixing the parameter types.